src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/TextBoxRendererTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Drawing;
+using System.Windows.Forms.VisualStyles;
+
+namespace System.Windows.Forms.Tests;
+
+public class TextBoxRendererTests
+{
+    [WinFormsTheory]
+    [InlineData(TextBoxState.Normal)]
+    [InlineData(TextBoxState.Hot)]
+    [InlineData(TextBoxState.Selected)]
+    [InlineData(TextBoxState.Disabled)]
+    public void TextBoxRenderer_DrawTextBox(TextBoxState state)
+    {
+        using Bitmap bitmap = new(100, 100);
+        using Graphics graphics = Graphics.FromImage(bitmap);
+        Rectangle bounds = new Rectangle(10, 20, 30, 40);
```

Comment:
Please use target type new.
```suggestion
        Rectangle bounds = new(10, 20, 30, 40);
```

Same with some other places in this file as well.